sdk/keyvault/keyvault-keys/tests-resources.json

```diff
@@ -51,16 +51,7 @@
                 "verify",
                 "wrapKey"
               ],
-              "secrets": [
-                "backup",
-                "delete",
-                "get",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "set"
-              ],
+              "secrets": ["backup", "delete", "get", "list", "purge", "recover", "restore", "set"],
```

Comment:
Nit: IMO, I'd leave the line breaks since it makes diff'ing/merging easier later.

Reply:
this was done automatically, I'll undo it

Reply:
we should file an issue to revisit our prettier settings.